src/CampaignsAdmin/DropPhotoZone.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { NativeTypes } from 'react-dnd-html5-backend';
+import { useDrop, DropTargetMonitor } from 'react-dnd';
+import { InboxOutlined, DownloadOutlined } from '@ant-design/icons';
+import './dropPhotoZone.css';
+
+interface TargetBoxProps {
+  onDrop: (item: { files: File[] }) => void;
+}
+export const DropPhotoZone: FC<TargetBoxProps> = (props:TargetBoxProps ) => {
+  const { onDrop } = props;
+  const [{ canDrop, isOver }, drop] = useDrop(
+    () => ({
+      accept: [NativeTypes.FILE],
+      drop(item: { files: File[] }) {
+        if (onDrop) {
+          onDrop(item);
+        }
+      },
+      collect: (monitor: DropTargetMonitor) => ({
+        isOver: monitor.isOver(),
+        canDrop: monitor.canDrop(),
+      }),
+    }),
+    [props],
```

Comment:
use the specific dependencies here instead. `props` might not change since you're pointing to the reference of the whole object (or might change completely because new params always getting passed) .

Reply:
I don't know what other dependence use :/

Reply:
`props.onDrop` es lo unico que necesitas aqui. 

Si tienes eslint agrega la regla `react-hooks/exhaustive-deps` y VsCode te dice directo que necesitas